### DISCUSSION TITLE
Possible to mark tabs, not urls?

### DISCUSSION THREAD

**Discussion by rieje** [gdh1995/vimium-c#896] [Q&A]
Is it possible to mark tabs and not URLs? Particularly useful for those with hundreds of tabs where I want to jump to a certain part of the long list of vertical tabs (e.g. a global mark at beginning of tab list, another at another location). Currently, I global marks often opens a new tab to an old URL marked because I had deleted that tab.

**Comment by gdh1995** [comment]
What do you want to do, if a marked tab is not existing on `Marks.activate`?

Up to now you may write `$then` like this:
```
map ` Marks.activate $else="on_no_tab"
map <v-on_no_tab> showHUD text="marked-tab-is-not-found."
```
